Content.Client/Guidebook/GuidebookWindow.Layout.xaml.cs

```diff
@@ -0,0 +1,197 @@
+                    break;
+                }
+
+                parentStack.Add(new GridContainer()
+                {
+                    HorizontalExpand = true,
+                    Columns = int.Parse(args[1]),
+                    HorizontalAlignment = args.Length >= 3
+                        ? Enum.Parse<HAlignment>(args[2])
+                        : HAlignment.Left,
+                });
+
+                break;
+            }
+            case "{|":
+            {
+                if (args.Length > 3)
+                {
+                    Logger.Warning($"`{{|` directive didn't get expected arguments, see {directive}");
+                    break;
+                }
+
+                parentStack.Add(new BoxContainer()
+                {
+                    HorizontalExpand = true,
+                    Orientation = args.Length >= 2
+                        ? Enum.Parse<BoxContainer.LayoutOrientation>(args[1])
+                        : BoxContainer.LayoutOrientation.Horizontal,
+                    HorizontalAlignment = args.Length >= 3
+                        ? Enum.Parse<HAlignment>(args[2])
+                        : HAlignment.Left,
+                });
+
+                break;
+            }
+            case "|}":
+            {
+                if (args.Length != 1)
+                {
+                    Logger.Warning($"`|}}` directive didn't get expected arguments, see {directive}");
+                    break;
+                }
+
+                if (parentStack.Count == 1)
+                {
+                    Logger.Warning("Tried to remove the guide container from the parent stack. Do you have an extra `?}`?");
+                    break;
+                }
+
+                control = parentStack.Pop();
+                return;
+            }
+            case var _ when args[0].StartsWith("embedEntity"):
+            {
+                if (args.Length != 2 && args.Length != 3 && args.Length != 4)
+                {
+                    Logger.Warning($"`embedEntity` directive didn't get expected arguments, see: {directive}");
+                    break;
+                }
+
+                var ent = args[1];
+                var scale = args.Length >= 2 ? float.Parse(args[2]) : 1.0f;
+                control = new GuideEntityEmbed(ent, args[0].Contains("Caption"), args[0].Contains("Interactive"))
+                {
+                    HorizontalAlignment = HAlignment.Center,
+                    Scale = new Vector2(scale, scale),
+                    HorizontalExpand = args.Length >= 4 ? bool.Parse(args[3]) : false,
+                };
+                return;
+            }
+            case "controlsButton":
+            {
+                var button = new Button()
+                {
+                    Text = Loc.GetString("ui-info-button-controls"),
+                };
+                button.OnPressed += _ => new OptionsMenu().Open();
+                control = button;
+                return;
```

Comment:
Hard-coding a directive to open a specific button, which will probably only ever be used in a single guidebook feels very icky.

Maybe make it a general button to run client-side commands, and create a `openoptions` command (if one doesn't already exist?)